### ISSUE TITLE
Consider changing width of card when desktop full screen

### ISSUE THREAD

**Issue by bobbykwong** [sei22-project/bobby#57]
Card now stretches to max width for all user screens. This may not look so nice for desktop view. Might want to consider making it a bit smaller for desktop

**Comment by zachariahchow** [comment]
I think as a stopgap, this is handled easily in tailwind using breakpoints. try adding max-w-screen-lg or max-w-screen-xl?

**Comment by bobbykwong** [comment]
yeah, I was just thinking whether you guys want the current design to stretch or change it to have the breakpoints
